src/app/core/entity-components/entity-utils/dynamic-form-components/edit-single-entity/edit-single-entity.component.html

```diff
@@ -1,21 +1,32 @@
 <mat-form-field *ngIf="!selectedEntity || editingSelectedEntity" [formGroup]="formControl.parent">
   <mat-label>{{ label }}</mat-label>
+  <!--
+    This input is need for detecting errors in the form.
+    The actual input used for editing is below.
```

Comment:
Because we cannot connect the `formControl` for the default form (this one is only using the entity ID's) with the autocomplete input (which uses the whole entities) but we need a `formControl` inside the `<mat-form-field>` to work properly with validation and error messages.